csrc/mmdeploy/codebase/mmpose/topdown_affine.cpp

```diff
@@ -70,9 +76,13 @@ class TopDownAffine : public transform::Transform {
 
       cv::warpAffine(src, dst, trans, {image_size_[0], image_size_[1]}, cv::INTER_LINEAR);
     } else {
-      cv::Mat trans =
-          GetAffineTransform({c[0], c[1]}, {s[0], s[1]}, r, {image_size_[0], image_size_[1]});
-      cv::warpAffine(src, dst, trans, {image_size_[0], image_size_[1]}, cv::INTER_LINEAR);
+      if (use_warp_affine_) {
+        cv::Mat trans =
+            GetAffineTransform({c[0], c[1]}, {s[0], s[1]}, r, {image_size_[0], image_size_[1]});
+        cv::warpAffine(src, dst, trans, {image_size_[0], image_size_[1]}, cv::INTER_LINEAR);
+      } else {
```

Comment:
I think the problem is `TopDownAffine` is slow when `data["rotation"].get<float>` is 0

Reply:
I think the operation that takes the most time in `TopDownAffine` is `cv::warpAffine`, and the `rotation` has no effect since the warpAffine algorithm doesn't care about the values of the trans matrix